stoservice/src/main/java/com/tutrit/stoservice/controller/OrderController.java

```diff
@@ -0,0 +1,31 @@
+package com.tutrit.stoservice.controller;
+
+import com.tutrit.stoservice.bean.Order;
+import com.tutrit.stoservice.service.OrderService;
+
+public class OrderController implements CommandController {
+
+    private static final Command command = Command.REGISTER_NEW_ORDER;
+    private OrderService orderService;
+
+    public void setOrderService(OrderService orderService) {
+        this.orderService = orderService;
+    }
+
+    public OrderController(OrderService orderService) {
+        this.orderService = orderService;
+    }
```

Comment:
you either have a mandatory dependency that must arrive through the constructor + the field must be final, or variable dependencies through a setter